content/compose/compose-file/05-services.md

```diff
@@ -1407,23 +1407,26 @@ expressed in the short form.
 - `published`: The publicly exposed port. It is defined as a string and can be set as a range using syntax `start-end`. It means the actual port is assigned a remaining available port, within the set range.
 - `host_ip`: The Host IP mapping, unspecified means all network interfaces (`0.0.0.0`).
 - `protocol`: The port protocol (`tcp` or `udp`). Defaults to `tcp`.
+- `app_protocol`: The application procotol (TCP/IP level 4 / OSI level 7) this port is used for. This is optional and can be used as a hint for Compose to offer richer behavior for protocols that it understands.
```

Comment:
suggestion:
```suggestion
- `app_protocol`: Layer 7 (application layer) protocol this port is used for. This is optional and can be used as a hint for Compose to offer richer behavior for protocols that it understands.
```

Reply:
question: Should we list valid options? Is there an enum of valid options like `http|https|ssh|ftp|...`